src/types/types.tsx

```diff
@@ -0,0 +1,89 @@
+export interface Coin {
```

Comment:
It's really better to place types close to the components/functions they are related to instead of one huge types file. Also, please make the types files extension `.ts`